torch/_dynamo/guards.py

```diff
@@ -722,6 +732,7 @@ def compile_check_fn(
                 ("___check_tensors", check_tensors_fn),
                 ("___check_tensors_verbose", check_tensors_verbose_fn),
                 ("tensor_check_names", tensor_check_names),
+                ("oo", float("inf")),
```

Comment:
Nope, figure out how to not do this. infinity should never be dumped in the guard code.

Reply:
ack